datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -3338,6 +3373,11 @@ type GlossaryTermAssociation {
     The glossary term itself
     """
     term: GlossaryTerm!
+
+    """
+    Reference back to the associated urn for tracking purposes e.g. when sibling nodes are merged together
+    """
+    associatedUrn: String!
```

Comment:
Question - why wouldn't we call this  "sourceUrn"???

"associatedUrn" inside an _association_ can be confusing. If we think of it in a source -> edge -> dest model like we do elsewhere its clearer